functions/wifi.sh

```diff
@@ -18,7 +18,7 @@ wifi_setup() {
     SSID="myWifiSSID"
     PASS="myWifiPassword"
   fi
-  if is_pithree || is_pithreeplus; then cond_redirect apt -y install firmware-brcm80211; fi
+  if is_pithree || is_pithreeplus; then cond_redirect apt -y install firmware-brcm80211; rfkill unblock wifi; ifconfig wlan0 up; fi
```

Comment:
At this point we can break the block into multiple lines.

Reply:
here would also be a new if-loop with check against `is_pithreeplus` be better i think

Reply:
@ThomDietrich If you are able to parse the result of `rfkill list`, you would be able to check the existing rfkill setting and proceed with the next step, irrespective of the availability of the `is_pithreeplus` variable.
I checked this command on my other RBP3B and that results in:
`[09:34:29] openhabian@openHABianPi:~$ rfkill list`
`0: phy0: Wireless LAN`
`	Soft blocked: no`
`	Hard blocked: no`

Reply:
On some systems the command might not even be available. On my RPi3 (the old one) with Jessie:

```
-bash: rfkill: command not found
```

I agree with @haroldh. Let's check for rfkill first, then check it's `list` output, then unblock.

Reply:
@ThomDietrich The version of RFKILL on both my RBP's is `rfkill 0.5-1 (Raspbian)`. And as they are both built by Openhabian, I am expecting that UTIL-LINUX (RFKILL is part of this) is included in both Jessie and Stretch.